standard/statements.md

```diff
@@ -1308,6 +1314,80 @@ The order in which `foreach` traverses the elements of an array, is as follows:
 >
 > *end example*
 
+#### §async-foreach await foreach
+
+The compile-time processing of a `foreach` statement first determines the ***collection type***, ***enumerator type*** and ***iteration type*** of the expression. The processing for a `foreach` statement is detailed in §sync-foreach and the process for an `await foreach` is detailed in §async-foreach.
```

Comment:
This is stated in 1115 & 1123. Revising the split of material between General, the sync, and the async subclauses might improve matters.

Reply:
I think just deleting this paragraph would be fine - there may be more split revision to do, but I *think* in this case it's just a matter of removing the paragraph here and at 1123. We'd need to replace "This determination" with "The determination of the ***collection type***, ***enumerator type*** and ***iteration type*** of the expression" as well though (in both places).